pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "desec-dns"
+version = "0.1.0"
+description = "CLI for desec.io's http api."
+packages = [{ include = "desec.py" }]
+authors = ["Sebastian Hamann <code@ares-macrotechnology.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+requests = ">=2.0.0"
+cryptography = { version = ">=1.8", optional = true }
+dnspython = { version = ">=2.0.0", optional = true }
+
+[tool.poetry.extras]
+tlsa = ["cryptography"]
+zonefiles = ["dnspython"]
+all = ["cryptography", "dnspython"]
+
+[tool.poetry.group.dev.dependencies]
+flake8 = "^7.0.0"
```

Comment:
Please remove the dependency on flake8.

Reply:
Nevermind about `flake8`, I just saw that you already introduced a `pyproject.toml` with config for ruff, black and mypy. 9938068f4528dbf2d7d064ade09b6c76bfeff5f0 adds as dev dependencies accordingly.

Reply:
Great, thanks! However, please remove black. Ruff should be enough. Black is only configured for the benefit of people who prefer it over ruff.